src/QirRuntime/test/QIR-tracer/generate.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
This is not automation but a local tool to create test input that is then checked in. I cannot switch to using Q# project for the tracer yet, because it relies on custom targeting that we still need to figure out how to integrate with the compiler.